src/SessionSetup/include/SessionSetup/LoadCaptureWidget.h

```diff
@@ -15,14 +16,15 @@
 #include <memory>
 
 #include "CaptureFileInfo/ItemModel.h"
+#include "SessionSetup/HasRadioButtonInterface.h"
 
 namespace Ui {
 class LoadCaptureWidget;  // IWYU pragma: keep
 }
 
 namespace orbit_session_setup {
 
-class LoadCaptureWidget : public QWidget {
+class LoadCaptureWidget : public QWidget, HasRadioButtonInterface {
```

Comment:
note that `HasRadioButtonInterface` is not a public base class in this case. But I don't think you want private inheritance.

Reply:
No there is nothing that you are missing. I remove the interface completely